Mixpanel/Mixpanel.h

```diff
@@ -201,6 +201,18 @@
 
 /*!
  @property
+ 
+ @abstract
+ Controls whether to allow the lib to make connections to the Mixpanel
+ socket server regardless of the state of the DEBUG flag.
+ 
+ @discussion
+ Defaults to YES.
+ */
+@property (atomic) BOOL allowSocketConnection;
```

Comment:
Is there any way we can unify this and the `DISABLE_MIXPANEL_AB_DESIGNER` macro. 

We use that macro to effectively disable the designer connection by not adding the 4-fingers gesture recognizer. My concern is that now we have 4 possible combinations of this flag and the macro to think about. 

The reason it's a macro is that I figured that would be easier for people to add it to only their production build target.